.gitignore

```diff
@@ -7,6 +7,7 @@ cmd/VERSION
 *~
 *.swp
 .vscode/
+.idea/
```

Comment:
(nitpick^2) a bit unrelated to this PR but I understand that it's a bit silly to open a separate PR just for this change